stdlib/ctypes/__init__.pyi

```diff
@@ -194,7 +194,7 @@ class _SimpleCData(Generic[_T], _CData):
 class c_byte(_SimpleCData[int]): ...
 
 class c_char(_SimpleCData[bytes]):
-    def __init__(self, value: int | bytes = ...) -> None: ...
+    def __init__(self, value: int | bytes | bytearray = ...) -> None: ...
```

Comment:
I failed to trace through the code for this, but I trust this is correct.

Reply:
I didn't find the code either but tried it in the REPL. This one allows bytearray but `c_char_p` doesn't.